### PR TITLE
comma splice and ellipsis grammar fix

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Repukan** [comment]
How the hell is removing one dot a grammar error.

**Comment by ike709** [comment]
@Repukan An ellipsis is three dots...

**Comment by Armhulen** [comment]
^^^

**Comment by nicbn** [comment]
Use … instead.

**Comment by Armhulen** [comment]
does anything else in the code do that?

**Comment by nicbn** [comment]
Nevermind.

**Comment by praisenarsie** [comment]
better add that no points prb tag because arm is just too lazy to do fixes

**Comment by optimumtact** [comment]
It's a valid method of getting points

Think of it this way, if the system didn't exist, these things would never get done.

They improve the overall polish of the game

**Comment by nicbn** [comment]
I don't think these things would never get done
I mean I fixed some bugs before even knowing this system existed

**Comment by optimumtact** [comment]
you are not representative of the majority of contributors then

File: code/modules/mob/living/simple_animal/spawner.dm [hunk 1]
```diff
@@ -57,7 +57,7 @@
 
 /mob/living/simple_animal/hostile/spawner/skeleton
 	name = "bone pit"
-	desc = "A pit full of bones, some still seem to be moving.."
+	desc = "A pit full of bones, some still seem to be moving..."
```
Comment (duncathan) [inline code/modules/mob/living/simple_animal/spawner.dm]: while you're here fix the comma splice

**Review by duncathan — CHANGES_REQUESTED** [review]

**Comment by duncathan** [comment]
These changes absolutely should be encouraged by the PRB system.

**Pull request merged**